Update classifications (#22)
WayScience/cell-health-data: 4.classify-single-cell-phenotypes/4a.classify-single-cell-phenotypes/README.md

# 4a. Classify Single Cell Phenotypes

> In this module, we present our pipeline for classifying single cell phenotypes.

Reviewer: ```suggestion
In this module, we present our pipeline for classifying single-cell phenotypes.
```

> ## Deriving Classifications:

Reviewer: ```suggestion
## Deriving Classifications
```

Follow the steps in [4.classify-single-cell-phenotypes/README.md](../README.md) to run this module.

## File Description

> This module outputs compressed csv files containing the classification probabilities of preprocessed Cell-Health features output by [3.preprocess-features](../../3.preprocess-features/).

Reviewer 2: ```suggestion
This module outputs compressed CSV files containing the classification probabilities of preprocessed Cell-Health features output by [3.preprocess-features](../../3.preprocess-features/).
```

> This module outputs compressed csv files that contain the classification probabilities of preprocessed Cell Health features output by [3.preprocess-features](../../3.preprocess-features/).

Reviewer: ```suggestion
This module outputs compressed csv files containing the classification probabilities of preprocessed Cell-Health features output by [3.preprocess-features](../../3.preprocess-features/).
```

> These classifications are derived per-plate by each of the models created in the [phenotypic_profiling](https://github.com/WayScience/phenotypic_profiling_model) repository.

Reviewer: ```suggestion
These classifications are derived per plate by each of the models created in the [phenotypic_profiling](https://github.com/WayScience/phenotypic_profiling_model) repository.
```

> These models include multi-class models (16 output probabilities, 1 for each phenotypic class) and single-class models (2 output probabiltiies, positive and negative probabilities for the model's particular phenotypic class).

Reviewer: ```suggestion
These models include multi-class models (16 output probabilities, 1 for each phenotypic class) and single-class models (2 output probabilities, positive and negative probabilities for the model's particular phenotypic class).
```

For more information on model training and type see [phenotypic_profiling_model/2.train_model](https://github.com/WayScience/phenotypic_profiling_model/tree/main/2.train_model).

> Each compressed csv file output by this module contains rows of single-cell metadata (location, perturbation, etc) and feature classification probabilities, as derived by the particular model.

Reviewer: ```suggestion
Each compressed csv file generated by this module includes rows of single-cell metadata (location, perturbation, etc) along with feature classification probabilities, which is determined by the specific model used.
```
Dervied, for me, was a bit difficult to understand. Could be commonly used term the project. Feel free to reject!

Author: Although I use derive throughout the project, I think this still adds clarity. Thanks!

> Each compressed csv file generated by this module includes rows of single-cell metadata (location, perturbation, etc) along with feature classification probabilities, which is determined by the specific model used.

Reviewer 2: ```suggestion
Each compressed CSV file generated by this module includes rows of single-cell metadata (location, perturbation, etc) along with feature classification probabilities, which are determined by the specific model used.
```

> There is one compressed csv file for each of the nine plates in the Cell Health dataset.

Reviewer: ```suggestion
There is one compressed csv file corresponding to each of the nine plates in the Cell-Health dataset.
```

> There is one compressed csv file corresponding to each of the nine plates in the Cell-Health dataset.

Reviewer 2: ```suggestion
There is one compressed CSV file corresponding to each of the nine plates in the Cell-Health dataset.
```

> The contents of the compressed csv file containing `OutOfFocus` model predictions from plate `SQ00014613` are shown below:

Reviewer 2: ```suggestion
The contents of the compressed CSV file containing `OutOfFocus` model predictions from plate `SQ00014613` are shown below:
```

Reviewer 2: Might also add that this is an example of an output, but just a suggestion.

> | Location_Center_X | Location_Center_Y | Metadata_Site | Metadata_Well | Metadata_Plate | Metadata_Plate_Map_Name | Metadata_Reagent | OutOfFocus | OutOfFocus Negative |

Reviewer 3: Great documentation! Consider just including the table in a notebook, or reducing the number of example entries.

|-------------------|-------------------|---------------|---------------|----------------|--------------------------|------------------|------------|---------------------|
| 1141.205          | 27.372            | 4             | G18           | SQ00014613     | SQ00014613_G18_04        | ARID1B-2         | 0.123      | 0.877               |
| 896.493           | 42.108            | 4             | G18           | SQ00014613     | SQ00014613_G18_04        | ARID1B-2         | 0.194      | 0.806               |
| 1571.713          | 30.771            | 4             | G18           | SQ00014613     | SQ00014613_G18_04        | ARID1B-2         | 0.191      | 0.809               |
| ...          | ...            | ...             | ...           | ...     | ...        | ...         | ...      | ...               |
| 1535.734          | 55.748            | 9             | P21           | SQ00014613     | SQ00014613_P21_09        | PSMA1-1         | 0.385      | 0.615               |

## File Structure

> The output file structure of this module mirrors the structure of the models hosted at [phenotypic_profiling_model/2.train_model/models](https://github.com/WayScience/phenotypic_profiling_model/tree/main/2.train_model/models), with folders contianing plate classifications in place of the models.

Reviewer: ```suggestion
The output file structure of this module mirrors the structure of the models hosted at [phenotypic_profiling_model/2.train_model/models](https://github.com/WayScience/phenotypic_profiling_model/tree/main/2.train_model/models), with folders containing plate classifications in place of the models.
```

> The output structure is as follows:

Reviewer 3: Very comprehensive. This can be a good thing, but more information can also make it harder to follow or emphasize the most important info. Something to consider, but it's definitely a balance
